src/Core/Geometry/RGBDImage.cpp

```diff
@@ -57,7 +57,7 @@ RGBImagePyramid CreateRGBDImagePyramid(const RGBDImage& rgbd_image,
 			num_of_levels, with_gaussian_filter_for_depth);
 	RGBImagePyramid rgbd_image_pyramid;
 	rgbd_image_pyramid.clear();
-	for (int level = 0; level < num_of_levels; level++) {
+	for (size_t level = 0; level < num_of_levels; level++) {
```

Comment:
Good point. I think this lines
https://github.com/IntelVCL/Open3D/blob/9411b84e92d0994985632d564fbd4fac3e4af10a/src/Core/Odometry/Odometry.cpp#L450

need to be 
```
size_t num_levels = iter_counts.size();
```

Reply:
@syncle thank you for the review.
I think this line is fine, since it is converted to `int` explicitly.

Changing it to `size_t` requires changing the following line at the same time.
https://github.com/IntelVCL/Open3D/blob/9411b84e92d0994985632d564fbd4fac3e4af10a/src/Core/Odometry/Odometry.cpp#L466

But if `level` in the above line is changed to `size_t`, then it is always larger than or
equal to 0, since it is unsigned. Then the for loop never ends.